unfetter-discover/unfetter-ui
# Issue 862 and 866 (New button and screen for capability assessments)

Fixes unfetter-discover/unfetter#862
Fixes unfetter-discover/unfetter#866

Based on a stripped-down version of current Assessments, will be a space for developing UI 
 and related components for Capability Assessments.  Functionality of capability assessments will largely be modeled after current assessments, but most UI workflows will wait until ADG designs arrive.  Sub-components of these assessments will be most of the focus until then.

# To test:

1. Pull issue-866 branch
2. Restart unfetter-ui:  docker restart unfetter-ui
3. Note new "Assessments 3.0" button in navigator dropdown
4. First step of a new object assessment UI is displayed
